js/timers.ts

```diff
@@ -226,6 +233,8 @@ export function setTimeout(
   delay: number,
   ...args: Args
 ): number {
+  // @ts-ignore
```

Comment:
What error are you ignoring and why?

Reply:
Because I used `this` in a function(not a class)

Reply:
Possibly a helpful tip: you can put the reasoning for a `ts-ignore` on the same line:

example:
```ts
// @ts-ignore : because I used this in a function (not a class)
checkThis(this);
```

Reply:
Defining `this` in the arguments would remove this error.  Under strict mode, where `this` cannot be inferred, you have to be explicit.  It isn't an error to ignore.

```
function setTimer(
	this: unknown,
    cb: (...args: Args) => void,
	delay: number,
	...args: unknown[]
): number {
	checkThis(this);
	// ...
}
```